openapi.yaml

```diff
@@ -248,7 +248,7 @@ components:
               type: string
               format: uuid
               description: The unique identifier of the user who triggered the test.
-              example: "d61e8681-4715-40f2-8651-6d7d2c2de211"
+              example: "2e2bb27b-a19c-47ce-a9b6-cd1bd31622dc"
```

Comment:
this one is still the same as testtargetId above

Reply:
All are generated now